src/test/java/uk/gov/pay/api/filter/ratelimit/RateLimiterTest.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.pay.api.filter.ratelimit;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public class RateLimiterTest {
```

Comment:
This test can be got rid of if we use a real LocalRateLimiter and RedisRateLimiter in the RateLimiterFilterTest.

Reply:
Keeping it as discussed, as it is checking the switch between Local and Redis rate limiters.